src/sagas/CreateClaim.saga.test.ts

```diff
@@ -0,0 +1,179 @@
+import { Frost } from '@po.et/frost-client'
+import { delay } from 'redux-saga'
+import { takeLatest, call, put } from 'redux-saga/effects'
+import { describe } from 'riteway'
+const { toast } = require('react-toastify')
+
+import { Actions } from '../actions/index'
+import { CreateClaimSaga, handleOnCreateClaim, handleOnCreateClaimSuccess, handleOnCreateClaimError } from './CreateClaim.saga'
+import { WorkAttributes } from '../interfaces/Props'
+import { Configuration } from '../configuration'
+
+const createWork = ({
+  name = 'name',
+  dateCreated = new Date().toISOString(),
+  datePublished = new Date().toISOString(),
+  content = 'test',
+  tags = 'test',
+  author = 'author'
+}: WorkAttributes = {
+  name: 'name',
+  dateCreated: new Date().toISOString(),
+  datePublished: new Date().toISOString(),
+  content: 'test',
+  tags: 'test',
+  author: 'author'
+}) => ({
+  name,
+  dateCreated,
+  datePublished,
+  content,
+  tags,
+  author
+})
+
+describe('CreateClaimSaga Saga', async assert => {
+  const iterator = CreateClaimSaga()();
+  
+  assert({
+    given: 'Work Claim Form Submit Action',
+    should: 'handle post work request',
```

Comment:
Still a reference to post work request, this is now create claim. Same goes for the next few assertions.